app/javascript/controllers/projects/samples/attachments/concatenations_files_controller.js

```diff
@@ -0,0 +1,44 @@
+import { Controller } from "@hotwired/stimulus";
+
+// creates a table listing all the selected files
+export default class extends Controller {
+  static targets = ["field"];
+
+  connect() {
+    const table = document.getElementById(
+      "attachments-table-body"
+    ).parentElement;
+
+    const headers = table.getElementsByTagName("th");
+    const headersIndex = [];
+    for (let header of headers) {
+      if (["Filename", "Type", "Size"].includes(header.innerText)) {
```

Comment:
We probably should update this to just get the header with index `n, n1, n2, etc`. The reason for this is that if someone is using a french version of the site the headers will no longer match up with the list we have hardcoded here

Reply:
Like so? Please see 2450bc5965adf416a91e6591365de7afd5e79f18.